test/unit/specs/Todo.spec.ts

```diff
@@ -10,7 +10,7 @@ describe("Todo.vue", () => {
       }
     }).$mount();
 
-    const actual = vm.$el.textContent;
+    const actual = <string> vm.$el.textContent;
```

Comment:
textContent is `string | null` so we need to cast to `string` for passing to `assert.include`